ci/build_image.sh

```diff
@@ -1,42 +1,30 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
-source $(dirname $0)/env.sh
-
```

Comment:
env.sh isnt present in this repository, was it always erroring then? Just curious, nice to see external dependency removed

Reply:
The old gitlab version has the `env.sh` file.